Pod/Classes/Cards/Default Card Types/UnknownCardType.swift

```diff
@@ -15,18 +15,18 @@ public struct UnknownCardType: CardType {
     public let CVCLength = 0
     public let identifyingDigits: Set<Int> = []
 
-    public func validateNumber(cardNumber: Number) -> CardValidationResult {
+    public func validate(_ number: Number) -> CardValidationResult {
```

Comment:
This doesn't match the protocol signature `validate(number: Number)`. The effect is that there are now two functions for this class at the moment: `validate(number: Number)` and `validate(_ number: Number)`.